web/src/components/RawDataEditor/locales/en-US.ts

```diff
@@ -16,5 +16,5 @@
  */
 export default {
   'component.rawDataEditor.tip': "Don't support edit mode currently",
-  'component.rawDataEditor.title': 'Raw Data Editor',
+  'component.rawDataEditor.title': 'CodeMirror',
```

Comment:
May I ask why did not use `Raw Data Editor` here?  It seems that this is  easier to understand

Reply:
😂 but we don't have a uniform  name for now, `Create with Editor`/`Raw Data Editor`

Reply:
How about looking at this issue from another angle:
`Create with Editor` is an action that can arouse `CodeMirror`, and `Raw Data Editor` can be considered as a name we gave to `CodeMirror`. So we just need to unify the action name and the `CodeMirror` name.

for the `CodeMirror` name, `Raw Data Editor `is a better choice IMHO